packages/fxa-content-server/app/scripts/views/post_verify/cad_qr/get_started.js

```diff
@@ -8,21 +8,32 @@ import FlowEventsMixin from './../../mixins/flow-events-mixin';
 import FormView from '../../form';
 import Template from 'templates/post_verify/cad_qr/get_started.mustache';
 import preventDefaultThen from '../../decorators/prevent_default_then';
+import { MOZ_ORG_SYNC_GET_STARTED_LINK } from '../../../lib/constants';
 
 class GetStarted extends FormView {
   template = Template;
   viewName = 'get-started';
+  settingReminder = false;
 
   events = assign(this.events, {
-    'click #maybe-later-btn': preventDefaultThen('clickMaybeLater'),
+    'click #maybe-later-link': preventDefaultThen('clickMaybeLater'),
   });
 
   submit() {
     return this.navigate('/post_verify/cad_qr/ready_to_scan');
   }
 
-  clickMaybeLater() {
-    return this.navigate('/sms');
+  async clickMaybeLater() {
+    if (this.settingReminder) {
+      return;
+    }
+
+    this.settingReminder = true;
```

Comment:
Is the intention that `this.settingReminder` only allow the user to click the button once? I don't think this buys anything because the page does a navigate away and if the user clicks back, these values would be reloaded. It might be easier for us to rate limit on auth-server side of things.

Reply:
I agree about rate limiting, but depending on network speed you could rapidly click the link and create multiple requests. Probably super edge case but couldn't hurt?